### DISCUSSION TITLE
How to make background image transparent without affecting the content inside?

### DISCUSSION THREAD

**Discussion by Anindya088** [zauberzeug/nicegui#1133] [Q&A] [answered]
### Question

Hi All,

I am trying to create some content with a background image, and I would like to be able to modify the transparency of the background image. But I realized if I use ```opacity``` then the whole thing, meaning the image plus the content inside everything is fading out.

Here is what I tried:
```python
with ui.image('/images/bg.jpg').classes('opacity-20'):
    with ui.card():
        ui.label('User Form')
        ui.input('Username')
        ui.input('Password')
```
If I try the ```opacity``` class then the whole thing including the card and everything is fading out. Is there a way just to make the background image transparent but not the content inside?

Thanks,
Anindya

**Comment by falkoschindler** [comment, marked as answer]
Hi @Anindya088, this seems to be a [pretty difficult question](https://stackoverflow.com/questions/10422949/css-background-opacity).

But there's an easy alternative. Place the image inside the card using absolute positioning, zero inset (cover the complete card without padding) and an opacity of your choice:

```py
with ui.card().classes('bg-transparent'):
    ui.image('https://picsum.photos/id/112/640/360').classes('absolute inset-0 opacity-30')
    ui.label('User Form')
    ui.input('Username')
    ui.input('Password')
```

<img width="200" alt="Screenshot 2023-07-06 at 12 43 03" src="https://github.com/zauberzeug/nicegui/assets/5767091/c81df02b-9c90-4dde-91b0-166c11a9e01a">

**Reply by Anindya088** [reply]
Hi @falkoschindler , thanks so much, that's a very nice trick 😄 

I don't have any knowledge of CSS, so did not know this was a big thing. I was thinking probably somewhere in Tailwind there is a class or something which I can put inside the children objects, and then they can bypass the parent opacity, but seems like it does not work that way!

Thanks again for the workaround for this 😄 

Regards,
Anindya
